src/spatial.rs

```diff
@@ -23,27 +35,39 @@ pub struct AudioEmitter {
 #[derive(Component)]
 pub struct AudioReceiver;
 
-/// Configuration resource for spatial audio
+/// Configuration resource for global spatial audio radius
 ///
-/// If this resource is not added to the ECS, spatial audio is not applied.
+/// If neither this resource or the [`SpatialRadius`] component for entity exists in the ECS,
+/// spatial audio is not applied.
 #[derive(Resource)]
-pub struct SpatialAudio {
-    /// The volume will change from `1` at distance `0` to `0` at distance `max_distance`
-    pub max_distance: f32,
+pub struct GlobalSpatialRadius {
```

Comment:
This resource would make more sense as a field on `SpatialAudioPlugin`. E.g. `default_spatial_audio_radius`.